core/object/script_language_extension.cpp

```diff
@@ -61,14 +61,14 @@ void ScriptExtension::_bind_methods() {
 	GDVIRTUAL_BIND(_get_language);
 
 	GDVIRTUAL_BIND(_has_script_signal, "signal");
-	GDVIRTUAL_BIND(_get_script_signal_list);
+	GDVIRTUAL_BIND(_get_script_signal_list, "no_inheritance");
```

Comment:
Hm, I don't think we've dealt with compatibility for a change to the signature of a virtual method yet.

Old GDExtensions loaded into a newer version of Godot with this change will have their `_get_script_signal_list` function called with a parameter that they are not expecting. I'm not sure if this is safe to do.

This will need some more investigation and perhaps even testing on a few different platforms.